File: codeforces/contest69/A.java
import java.io.*;

public class A {
	public static int solve(int n) {
		int sum = 0;
		if (n == 1) {
			sum = 1;
		} else {
			sum = solve(n-1)*3;
		}
		return sum;
	}
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int n = Integer.parseInt(br.readLine());
		int ans = solve(n);
		System.out.println(ans);
	}
}

File: codeforces/contest69/A.c
#include <stdio.h>
#include <math.h>

int solve(int n) {
	int sum = pow(pow(2, n), 2)/2+pow(2, n)*2-1;
	if (n != 1) {
		int count = 1;
		while (count < n) {
			sum+=solve(count);
			count++;
		}
	}
	return sum;
}

int main() {
	int a;
	scanf("%d", &a);
	int ans = pow(pow(2, a), 2) - solve(a);
	printf("%d", ans);
}
